front/app/containers/Admin/projects/project/inputForm/IdeaFormBuilder/index.tsx

```diff
@@ -49,6 +56,7 @@ const IdeaFormBuilder = () => {
           goBackUrl,
           onDownloadPDF: handleDownloadPDF,
         }}
+        viewFormLink={`/projects/${project.data.attributes.slug}/ideas/new?phase_id=${phase.data.id}`}
```

Comment:
Why fetch the phase here, we already have the `phaseId` and it seems it's all that's needed.

Reply:
Yes, you are right. Not sure what went on in my mind there :D